gradle.properties

```
@@ -1,5 +1,5 @@
 kotlin.code.style=official
 kotlin_version =1.5.31
-release_version=4.6.1
+release_version=4.6.2
```

It was not a minor version. We add new functionality and we need to update the minor version instead of maintenance one
```suggestion
release_version=4.7.0
```